src/components/Navbar/Navbar.jsx

```diff
@@ -27,7 +27,7 @@ const Navbar = () => {
                         <span className="badge badge-sm center">{wishlist}</span>
                     </div>
                 </Link>
-                <Link to="/">
+                <Link to="/product">
```

Comment:
I hope you would be linking this `cart-icon` with `cart` route instead of `product` route